### PR TITLE
2754 show overlay footprint on hover

### PR DESCRIPTION
**Pull request by ramtob**

### CODE REVIEW

File: src/app/@ansyn/ansyn/modules/plugins/openlayers/plugins/visualizers/overlays/overlay-hover-visualizer.ts [hunk 1]
```diff
@@ -0,0 +1,181 @@
...
+
+	constructor(
+		public store$: Store<any>,
+		public actions$: Actions,
+		@Inject(VisualizersConfig) config: IVisualizersConfig,
+		public overlaysService: OverlaysService
+	) {
+		super(config.OverlayHoverVisualizer);
+		this.updateStyle({
+			opacity: 0.5,
```
Comment (asafMasa) [inline src/app/@ansyn/ansyn/modules/plugins/openlayers/plugins/visualizers/overlays/overlay-hover-visualizer.ts]: check style

File: src/app/@ansyn/ansyn/modules/plugins/openlayers/plugins/visualizers/overlays/overlay-hover-visualizer.ts [hunk 2]
```diff
@@ -0,0 +1,181 @@
...
+			const featureJson = feature(hoveredOverlay.footprint);
+			const entityToDraw = {id: UUID.UUID(), featureJson};
+			return this.setEntities([entityToDraw]);
+		} else {
+			this.clearEntities();
+			return of(true);
+		}
+	}
+
+	// Start: code partly shared with polyline-visualizer
```
Comment (asafMasa) [inline src/app/@ansyn/ansyn/modules/plugins/openlayers/plugins/visualizers/overlays/overlay-hover-visualizer.ts]: duplicated

File: src/app/@ansyn/ansyn/modules/plugins/openlayers/plugins/visualizers/overlays/overlay-hover-visualizer.ts [hunk 3]
```diff
@@ -0,0 +1,181 @@
...
+		switch (this.getFeatureType(feature)) {
+			case 'MultiLineString':
+			case 'MultiPolygon':
+				return hover ? 0.4 : 1;
+			case 'Point' :
+				return hover ? 1 : 0.8;
+			case 'LineString':
+				return undefined;
+			default:
+				return hover;
```
Comment (asafMasa) [inline src/app/@ansyn/ansyn/modules/plugins/openlayers/plugins/visualizers/overlays/overlay-hover-visualizer.ts]: check return value on default

File: src/app/@ansyn/ansyn/modules/plugins/openlayers/plugins/visualizers/overlays/polyline-visualizer.ts [hunk 4]
```diff
@@ -138,10 +112,7 @@ export class FootprintPolylineVisualizer extends BaseFootprintsVisualizer {
 	onSelectFeature($event) {
 		if ($event.selected.length > 0) {
 			const id = $event.selected[0].getId();
-			const hoverFeature = this.hoverLayer.getSource().getFeatureById(id);
-			if (!hoverFeature || hoverFeature.getId() !== id) {
```
Comment (asafMasa) [inline src/app/@ansyn/ansyn/modules/plugins/openlayers/plugins/visualizers/overlays/polyline-visualizer.ts]: check that you won't send same events over and over

File: src/app/@ansyn/ansyn/modules/plugins/openlayers/plugins/visualizers/overlays/polyline-visualizer.ts [hunk 5]
```diff
@@ -263,18 +200,14 @@ export class FootprintPolylineVisualizer extends BaseFootprintsVisualizer {
 		switch (this.getFeatureType(feature)) {
 			case 'MultiLineString':
 			case 'MultiPolygon':
+			case 'LineString':
 				return isActive ? active : isDisplayed ? display : inactive;
 			case 'Point' :
 				return display;
-			case 'LineString':
-				return isFavorites ? favorite : isActive ? active : display;
```
Comment (asafMasa) [inline src/app/@ansyn/ansyn/modules/plugins/openlayers/plugins/visualizers/overlays/polyline-visualizer.ts]: check is Favorite style

File: src/app/@ansyn/map-facade/components/imagery-status/imagery-status.component.ts [hunk 6]
```diff
@@ -69,7 +63,7 @@ export class ImageryStatusComponent implements OnInit, OnDestroy {
 		/* force angular to rerender the *ngFor content that binding to this arrays
 		 * so they get the new mapId	 */
 		this._entryComponents = { status: [], container: [], floating_menu: [] };
-		this.translate.get(this.overlay ? this.overlay.sensorName : 'sensorName')
```
Comment (asafMasa) [inline src/app/@ansyn/map-facade/components/imagery-status/imagery-status.component.ts]: unknown

File: src/app/@ansyn/ansyn/modules/plugins/openlayers/plugins/visualizers/overlays/overlay-hover-visualizer.ts (hunk 2 above)
Comment (ramtob) [reply src/app/@ansyn/ansyn/modules/plugins/openlayers/plugins/visualizers/overlays/overlay-hover-visualizer.ts]: done

File: src/app/@ansyn/ansyn/modules/plugins/openlayers/plugins/visualizers/overlays/overlay-hover-visualizer.ts (hunk 3 above)
Comment (ramtob) [reply src/app/@ansyn/ansyn/modules/plugins/openlayers/plugins/visualizers/overlays/overlay-hover-visualizer.ts]: done

File: src/app/@ansyn/ansyn/modules/plugins/openlayers/plugins/visualizers/overlays/polyline-visualizer.ts (hunk 5 above)
Comment (ramtob) [reply src/app/@ansyn/ansyn/modules/plugins/openlayers/plugins/visualizers/overlays/polyline-visualizer.ts]: done

File: src/app/@ansyn/map-facade/components/imagery-status/imagery-status.component.ts (hunk 6 above)
Comment (ramtob) [reply src/app/@ansyn/map-facade/components/imagery-status/imagery-status.component.ts]: done

File: src/app/@ansyn/ansyn/modules/plugins/openlayers/plugins/visualizers/overlays/polyline-visualizer.ts (hunk 4 above)
Comment (ramtob) [reply src/app/@ansyn/ansyn/modules/plugins/openlayers/plugins/visualizers/overlays/polyline-visualizer.ts]: Seem on-kay!

**Pull request merged**